CHANGELOG.md

```diff
@@ -18,7 +18,23 @@ and this project does not adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.1] - 2014-11-15
 ### Added
-- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1 . 
+- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1. 
+
+## [0.2] - 2017-01-25
```

Comment:
Please release with a tag the latest commit before the merge (i.e. the recovery branch). It could be either `0.1.1` or `0.2`, I do not have a strong opinion on this.